third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -631,6 +673,32 @@ func resourceComposerEnvironmentUpdate(d *schema.ResourceData, meta interface{})
 			}
 			d.SetPartial("config")
 		}
+
+    <% unless version == "ga" -%>
```

Comment:
super nitpicky, while you're here: the ERB tags shouldn't be indented, I think. We tend to indent tags relative to other tags while the Go code inside of them is indented relative to where it ends up in the final file.